src/basic/log.c

```diff
@@ -439,6 +445,9 @@ static int write_to_console(
         if (console_fd < 0)
                 return 0;
 
+        if (_likely_(LOG_PRI(level) > log_target_max_level[LOG_TARGET_CONSOLE]))
```

Comment:
The per target max level defaults to `INT_MAX`, so is this condition `_likely_` satisfied??